blueflood-elasticsearch/src/main/resources/metrics_mapping.json

```diff
@@ -13,7 +13,7 @@
                 "index": "not_analyzed"
             },
             "metric_name": {
-                "index_analyzer": "prefix-test-analyzer",
+                "analyzer": "prefix-test-analyzer",
                 "search_analyzer": "keyword",
```

Comment:
This change is needed because of the upgrade. Here is the link for that:
https://www.elastic.co/guide/en/elasticsearch/reference/2.0/breaking_20_mapping_changes.html#_analyzer_mappings

Regarding needed prod changes, I am assuming this is what you meant:
https://jira.rax.io/browse/CMD-1609